src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -71,43 +79,68 @@ def create_container(client,
                                                               username=registry_username,
                                                               password=registry_password)]
     elif ACR_SERVER_SUFFIX in image:
-        if registry_password is None:
+        if not registry_password:
             try:
                 registry_password = prompt_pass(msg='Image registry password: ')
             except NoTTYException:
                 raise CLIError('Please specify --registry-password in non-interactive mode.')
 
         acr_server = image.split("/")[0] if image.split("/") else None
         acr_username = image.split(ACR_SERVER_SUFFIX)[0] if image.split(ACR_SERVER_SUFFIX) else None
-        if acr_server is not None and acr_username is not None:
+        if acr_server and acr_username:
             image_registry_credentials = [ImageRegistryCredential(server=acr_server,
                                                                   username=acr_username,
                                                                   password=registry_password)]
         else:
             raise CLIError('Failed to parse ACR server or username from image name; please explicitly specify --registry-server and --registry-username.')
 
-    command = None
-    if command_line is not None:
-        command = shlex.split(command_line)
+    command = shlex.split(command_line) if command_line else None
+
+    azure_file_volume = None
+    if azure_file_volume_share_name:
+        if not azure_file_volume_account_name:
+            try:
+                azure_file_volume_account_name = prompt(msg='Azure File storage account name: ')
+            except NoTTYException:
+                raise CLIError('Please specify --azure_file_volume_account_name in non-interactive mode.')
```

Comment:
Should be `--azure-file-volume-account-name` if that's the actual argument name.
A user wouldn't know what `--azure_file_volume_account_name` is.